app/templates/bower.json

```diff
@@ -2,38 +2,43 @@
   "name": "@sample-app-name",
   "version": "0.0.0",
   "dependencies": {
-    "jquery": "~2.1.4",
     "angular": "~1.4.4",
+    "angular-animate": "~1.4.4",
     "angular-bootstrap": "^1.1",
     "angular-cookies": "~1.4.4",
+    "angular-google-maps": "2.3.2",
     "angular-highlightjs": "~0.4.3",
+    "angular-messages": "~1.4.4",
     "angular-mocks": "~1.4.4",
+    "angular-sanitize": "~1.4.4",
     "angular-ui-router": "~0.2.15",
     "angular-ui-tinymce": "~0.0.9",
-    "angular-animate": "~1.4.4",
-    "ngtoast": "^2.0.0",
     "angular-x2js": "https://github.com/janmichaelyu/angular-x2js.git",
     "bootstrap": "~3.3.5",
     "font-awesome": "~4.6.0",
+    "highcharts": "^4.2",
     "highlightjs":"~8.7.0",
+    "jquery": "~2.1.4",
```

Comment:
Thanks for alphabetizing this list. There is one exception though, jquery must be listed first (or at least before anything from Angular). That is to make sure Angular will use jquery, which is important for some of the reusable components we have put together..